README.md

```diff
@@ -76,18 +76,23 @@ You can find more examples in our [example page](https://dpp.dev/md_docpages_03_
 ## 💻 Supported Systems
 
 ### Linux
+
 The library runs ideally on **Linux**.
 
-### Mac OS X and FreeBSD
-The library is well-functional and stable on **Mac OS X** and **FreeBSD** too.
+### macOS and FreeBSD
+
+The library is well-functional and stable on **macOS** and **FreeBSD** too.
```

Comment:
technically only macOS X. macOS without the X implies that it also works on OS8, OS9 etc... which it doesnt as theyre really strange non-posix systems

Reply:
They changed the naming yet again few years back. No X in sight: https://support.apple.com/macos It used to be Mac OS, even used to be just OS X for a bit, but then they wanted to unify the scheme with iOS, so they made it just macOS.